app/modules/relationships/models.py

```diff
@@ -0,0 +1,116 @@
+# -*- coding: utf-8 -*-
+"""
+Relationships database models
+--------------------
+"""
+
+from sqlalchemy_utils import Timestamp
+
+from app.extensions import HoustonModel, db
+from datetime import datetime  # NOQA
+import app.extensions.logging as AuditLog
+
+import uuid
+
+import logging
+
+log = logging.getLogger(__name__)
+
+
+class RelationshipIndividualMember(db.Model, HoustonModel):
+
+    guid = db.Column(
+        db.GUID, default=uuid.uuid4, primary_key=True
+    )  # pylint: disable=invalid-name
+
+    relationship_guid = db.Column(db.GUID, db.ForeignKey('relationship.guid'))
+    relationship = db.relationship(
+        'Relationship', backref=db.backref('individual_members')
+    )
+
+    individual_guid = db.Column(db.GUID, db.ForeignKey('individual.guid'))
+    individual = db.relationship('Individual', backref=db.backref('relationships'))
+
+    individual_role = db.Column(db.String, nullable=False)
+
+    def __init__(self, individual, individual_role, **kwargs):
+        self.individual = individual
+        self.individual_role = individual_role
+        self.individual_guid = individual.guid
+
+
+class Relationship(db.Model, Timestamp):
+    """
+    Relationships database model.
+    """
+
+    guid = db.Column(
+        db.GUID, default=uuid.uuid4, primary_key=True
+    )  # pylint: disable=invalid-name
+
+    start_date = db.Column(
+        db.DateTime, index=True, default=datetime.utcnow, nullable=True
+    )
+    end_date = db.Column(db.DateTime, index=True, default=datetime.utcnow, nullable=True)
+
+    type = db.Column(db.String, nullable=True)
```

Comment:
old-world we also had a boolean for `bidirectional`.  do we need it here?  i cant find evidence of it really being used, other than simply reported as a property/value on the Relationship.   it seems like `type` + which `role` is assigned to which individual gets us the same thing.  (e.g. "bidirectional" type=familial, if role is `mother` and `calf`, it is implied by the types and roles that there is a direction.)

in summary:  i am all for dropping it; but just wanted to ask.

Reply:
Yeah, I think role assignment should be a more useful way of determining this. Hard to think of a relationship that _isn't_ bidirectional anyway.